apps/supernova/src/index.js

```diff
@@ -10,7 +10,17 @@ import React from "react"
 export const mount = (container, options = {}) => {
   import("./App").then((App) => {
     mount.root = createRoot(container)
-    mount.root.render(React.createElement(App.default, options?.props))
+    mount.root.render(
+      React.createElement(App.default, {
+        ...options?.props,
+        /**
+         * [DANGER] [DANGER] [DANGER] [DANGER]
+         * please remove this change before merging it to main
+         * this is only to see a working demo PR Preview
+         */
+        enableNewUrlStructure: true,
```

Comment:
[Self Reminder]: Don't forget to remove this after review.

Reply:
As I mentioned before we could enable here the new structure comparing with a query parameter for the preview.